ml-fundamentals/README.md

```diff
@@ -10,15 +10,23 @@ This guideline documents the Machine Learning (ML) practices in CSE. CSE works w
 
 ## How to use these fundamentals
 
-If you are starting a new ML project, consider reading through the [fundamentals checklist](ml-fundamentals-checklist.md) or [Responsible AI](https://www.microsoft.com/en-us/ai/responsible-ai-resources) links.
+* If you are starting a new ML project, consider reading through the [general guidance documents](#general-guidance).
+* For specific aspects of an ML project, refer to the guidelines for different [project phases](#ml-project-phases).
 
-For specific aspects of an ML project, refer to the guidelines for different [project phases](#ml-project-phases).
+## ML Project phases
 
-## Sections
+![Project flow](flow.png)
 
-### General
+* Envisioning: Initial problem understanding, customer goals and objectives.
+* Feasibility study: Assess whether the problem in question is feasible to solve satisfactorily using ML with the available data.

```

Comment:
Do we assume we already have the data here? Or, if we do not, do we identify what data we'd need and how we'd collect it in order to move forward?

Reply:
There will be a full section on feasibility study. During this study we first verify we have access to data. Do you think we should add clarification here as well?